src/input_file.rst

```diff
@@ -1182,8 +1182,8 @@ Note that some tags, or entries for tags, are only used in certain ensembles (e.
     # Mol.  Name Fugacity (bar)
     #################################
     Fugacity  ISB   10.0     12.00     14.00     16.00
-    Fugacity  Si     0.0       1.0       2.0       3.0
-    Fugacity  O      0.0       0.5      0.75       1.0
+    Fugacity  Si     0.0
+    Fugacity  O      0.0
```

Comment:
Still we should put zero for other Replicas. Please tell me that your code would catch this type of mistake and generate error.

Reply:
Either 1 value or the same amount are valid.  It would be preferable for the user to provide 1 so the folder names are shorter.  If the user includes more than more than 1 value, the values are used to create the name of the folder.  GOMC exits with an error if the values are of any length other than 1 or max(number of values provided across residues)

Reply:
Then you need to explain it in the Manual. How user need to find out this behavior?

Reply:
![image](https://user-images.githubusercontent.com/39970712/72471582-ec82fa80-37b0-11ea-8074-a2c16c090748.png)

Reply:
![image](https://user-images.githubusercontent.com/39970712/72471681-22c07a00-37b1-11ea-8324-f80f356102b3.png)

Reply:
Why there is 2 Note with the same message?  Just keep the last one. Please read it and make sure it make sense. 
.. note:: 
    - For binary systems, include multiple copies of the tag (one per residue kind).
    - If there is a molecule kind that cannot be transfer between boxes (in PDB file the beta value is set to 1.00 or 2.00) an arbitrary value e.g. 0.00 can be assigned to the residue name.

Reply:
You can merge your two Note in One, since they are related to MultiSim.

Reply:
I dont understand the note.  Is the zero supposed to be assigned to Residue name?  IT is assigned to value in the manual.

Reply:
Also, I wanted to make sure the user read the note in case they didnt want to use MPI and skipped the section.

Reply:
> I dont understand the note. Is the zero supposed to be assigned to Residue name? IT is assigned to value in the manual.

It's not zero, its Oxygen.

Reply:
In the note, it says Residue Name.

Reply:
![image](https://user-images.githubusercontent.com/39970712/72473883-a54b3880-37b5-11ea-8a64-185358a165db.png)

Reply:
> I dont understand the note. Is the zero supposed to be assigned to Residue name? IT is assigned to value in the manual.

If a molecule is fixed in its position or not supposed to leave the box, use must define it in the PDB file, and give a zero or any arbitrary number.  It's important that user sets a number, otherwise program exits with error.

Reply:
assign zero value as the fugacity to the the resname.

Reply:
So the zeroes assigned to Si and O are not examples of the note.  Because the residue name is assigned Si and O, as opposed to zero.

Reply:
Resname cannot be assign in the config file. The zero fugacity is assigned to the specific resname (Si, or O).

Reply:
It is confusing, I would say assigned to the residue fugacity value.

Reply:
Sure, you can modify the text.